src/plugins/intel_gpu/src/kernel_selector/kernels/resample/resample_kernel_opt.cpp

```diff
@@ -81,6 +81,22 @@ DeviceFeaturesKey ResampleKernelOpt::get_required_device_features_key(const Para
     return get_common_subgroups_device_features_key(params, options);
 }
 
+static size_t get_vec_size(const resample_params &params) {
+    if (params.inputs[0].GetLayout() == DataLayout::fs_b_yx_fsv32) {
+        return 2;
+    } else {
+        return 1;
+    }
+}
+
+static int get_feature_slice_size(const resample_params &params) {
+    if (params.inputs[0].GetLayout() == DataLayout::fs_b_yx_fsv32) {
+        return 32;
+    } else {
+        return 16;
+    }
+}
```

Comment:
I think it would be better to use `get_vec_size` here? These two functions are similar.